src/encoded/audit/file.py

```diff
@@ -148,8 +148,8 @@ def audit_file_assembly(value, system):
     if 'derived_from' not in value:
         return
     for f in value['derived_from']:
-        if 'assembly' in f:
-            if f['assembly'] != value['assembly']:
+        if f.get('assembly') and value.get('assembly') and \
```

Comment:
it resolves audit error caused by my assumption that a file hat is derived from something will always have an assembly (turns out to be sometimes untrue)